### DISCUSSION TITLE
Saved Filters

### DISCUSSION THREAD

**Discussion by harryjms** [community/community#47220] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

The filtering for Pull Requests is very powerful and I use it a fair amount to narrow down and find a PR. I save my filters as bookmarks in chrome at the moment, but would love the ability to save them in github itself, as pills under the filter input perhaps.

**Comment by norwd** [comment]
This already exists in the notifications area (thanks goodness since I can filter out my 500+ @dependabot notifications) but I desperately need this in the PR list for some of our mono repos where the PRs are often in the hundreds...

**Comment by adambirds** [comment]
This 100% needs to be a thing.

**Comment by coatesap** [comment]
An alternative, which might be just as useful, would be if the filters persisted rather than reset on every page load. As long as there was a obvious 'reset filters' button or similar. 

Often I set some filters (author, or whatever), click into a PR, then click on PRs link, and all the filters are lost.

**Comment by ldavison-sg** [comment]
It would also be useful on the "repositories" page, especially for larger organisations. There are predefined filters (public, internal, etc); being able to save a custom filter for topics would be useful to us.

**Comment by lorenzo-attuned** [comment]
I just noticed the custom Views you can save in the [Issues section](https://github.com/issues/) can actually search for PRs too.

<img width="781" height="520" alt="Screenshot 2025-08-08 at 16 35 30" src="https://github.com/user-attachments/assets/24b403c3-ea6d-4774-b5f3-a29721d97af0" />
